src/components/UserManagement/UserOrderHistory/UserOrderHistory.tsx

```diff
@@ -103,7 +120,23 @@ export default class UserOrderHistory extends React.Component<Props, State> {
     // );
   }
 
+  submitSearch = async (evt) => {
+    if (evt.key === "Enter") {
```

Comment:
We should have this check inside of the search component.
That way, we won't have to re-implement it wherever we need to use the search bar.